### PR TITLE
Upgrade Zeebe to 8.1.1

### PR DESCRIPTION
**Pull request by arkadasfynarfin**
## Description

This PR upgrades the zeebe client version to 8.1.1

[GOV-449](https://mifosforge.jira.com/browse/GOV-449)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

[GOV-449]: https://mifosforge.jira.com/browse/GOV-449?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by arkadasfynarfin** [comment]
@fynmanoj Please review the PR.

File: build.gradle [hunk 1]
```diff
@@ -36,7 +36,8 @@ dependencies {
     implementation 'org.apache.camel.springboot:camel-jackson-starter:3.12.0'
     implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.13.1'
     implementation 'org.json:json:20211205'
-    implementation 'io.camunda:zeebe-client-java:1.3.1'
+    implementation 'io.camunda:zeebe-client-java:8.1.1'
+    implementation 'org.apache.camel:camel-http:3.12.0'
```
Comment (fynmanoj) [inline build.gradle:40]: why this line is showing in your change? is this dependency(org.apache.camel:camel-http) needed?

**Review by fynmanoj — CHANGES_REQUESTED** [review]
Clarification needed : org.apache.camel:camel-http

File: build.gradle (hunk 1 above)
Comment (arkadasfynarfin) [reply build.gradle]: @fynmanoj This is not a part of this change. This was added by mistake when resolving merge conflicts. This has been removed.

**Review by fynmanoj — APPROVED** [review]
LGTM

**Pull request merged**